src/styles/LoginStyles.ts

```diff
@@ -1,29 +1,31 @@
 import styled from 'styled-components';
 import { Button, TextField, Typography, Link, Container } from '@material-ui/core';
 
-export const LoginContainer = styled(Container)`
-    margin-top: 4%;
+export const MainDiv = styled('div')`
+    margin-top: 80px;
+`;
+
+export const FieldDiv = styled('div')`
+
 `;
 
 export const Label = styled(Typography)`
     color: #828282;
     font-size: 12px;
     font-weight: bold;
-    text-align: left;
-    padding: 6px 0px;
-    margin-left: 8%;
-    display: block;
+    margin-top: -20px;
+    margin-left: -83%;
```

Comment:
You can probably achieve a more accurate alignment with the FieldDev wrapper component above than this though. Try
```
export const FieldDiv = styled('div')`
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    width: 95%;
    margin: auto;
`;
```
to wrap the Label and Field together, aligning them to the left, then centering the wrapper div on the screen